File: 251110/특정 원소들의 합/sum-of-specific-elements.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n');
const grid = input.map(row => row.split(' ').map(Number));

let result = 0;
for (let i = 0; i < grid.length; i += 1) {
    for (let j = 0; j <= i; j += 1) {
        result += grid[i][j];
    }
}
console.log(result);

File: trail0/특정 원소들의 합/sum-of-specific-elements.cpp
#include <iostream>
using namespace std;

int main() {
    int grid[4][4], result = 0;
    for (int r = 0; r < 4; r++) {
        for (int c = 0; c < 4; c++) {
            cin >> grid[r][c];
        }
    }
    for (int r = 0; r < 4; r++) {
        for (int c = 0; c <= r; c++) {
            result += grid[r][c];
        }
    }
    cout << result;
    return 0;
}